docs/sources/http_api/licensing.md

```diff
@@ -0,0 +1,98 @@
++++
+title = "Licensing HTTP API "
+description = "Grafana Licensing HTTP API"
+keywords = ["grafana", "http", "documentation", "api", "licensing", "enterprise"]
+aliases = ["/docs/grafana/latest/http_api/licensing/"]
++++
+
+> Licensing is only available in Grafana Enterprise. Read more about [Grafana Enterprise]({{< relref "../enterprise" >}}).
+
+## Manually force license refresh
+
+> **Note:** Only available in Grafana Enterprise v7.4+.
```

Comment:
We are trying to avoid "only" in these notes, because it can cause confusion. Please remove that word in all cases and just say, "Available in..."